tests.py

```diff
@@ -76,7 +76,7 @@ def testParseArguments(self):
 
   def testDetectStorageFail(self):
     """Tests that the DockerExplorer.DetectStorage function fails on
-    non-existing Docker directory."""
+    Docker directory."""
```

Comment:
You removed something that seemed correct?

Reply:
I blame Git =(